apps/flame-defi/app/features/evm-wallet/components/connect-evm-wallet-button/connect-evm-wallet-button.tsx

```diff
@@ -84,15 +85,23 @@ export default function ConnectEvmWalletButton({
         <AccordionContent>
           <div className="text-white ml-8 flex justify-between">
             <div>
-              {isLoadingEvmNativeTokenBalance && <div>Loading...</div>}
-              {!isLoadingEvmNativeTokenBalance && evmNativeTokenBalance && (
+              <Skeleton
+                className="w-[150px] h-[20px] mb-2"
+                isLoading={
+                  isLoadingEvmNativeTokenBalance && !evmNativeTokenBalance
```

Comment:
```suggestion
                  Boolean(isLoadingEvmNativeTokenBalance && !evmNativeTokenBalance)
```

Reply:
This one isn't actually needed since the return type of `!evmNativeTokenBalance` is a boolean, but I kinda like being explicit.